modules/thunderlinkModule.js

```diff
@@ -121,18 +122,48 @@ function replaceVariables(template, hdr) {
   protectedSubject = protectedSubject.replace(/[<>'"`´]/g, "");
 
   // convert escape characters like \t to tabs
-  template = JSON.parse("\"" + template + "\"");
```

Comment:
I'm having difficulty with the JSON.parse method - if you try the "new" default of "clickable Mail.app link" it uses a template that has `"` characters in it, which doesn't work in JSON.parse. I tried a trivial workaround of escaping those prior and that didn't work because it generated 'local cmd' thunderlinks with extra `=` characters.

What can we do to make this one work?

This was actually a problem in the previous PR, I just didn't notice it until now

Reply:
We need a fix that doesn't require template changes, the installed base is more important than code elegance. A simple replace that escapes the equals sign before json parse here, leaving templates alone should work?

Reply:
Equals character was error in code. This is not a problem.

Regarding double quote sign, ok, I will change so that we have backward compatibility.